### ISSUE TITLE
Project:description is not loading

### ISSUE THREAD

**Issue by hguerra** [TerraME/terrame#1534]
```lua
import("terralib")

local proj = Project{
    file = "sampa.tview",
    title = "My title",
    description = "My description",
    clean = true,
    sampa = filePath("test/sampa.shp", "terralib")
}

print(proj.title)
print(proj.description)

proj = Project{
    file = "sampa.tview"
}

print(proj.title)
print(proj.description) -- Empty string.
```

**Comment by avancinirodrigo** [comment]
@pedro-andrade-inpe,
It seems to me that there is no test that check if description works #1115, so maybe it never works.

**Comment by pedro-andrade-inpe** [comment]
This should be implemented in #1115, but it wasn't. Can .tview files store that, @avancinirodrigo?
